mbed_host_tests/__init__.py

```diff
@@ -57,6 +57,9 @@
 HOSTREGISTRY.register_host_test("wait_us_auto", WaitusTest())
 HOSTREGISTRY.register_host_test("dev_null_auto", DevNullTest())
 
+# Set the default baud rate
+DEFAULT_BAUD_RATE = 115200
```

Comment:
I realize this default values are usually a heated topic, so I don't want to beat this to death. But all targets in mbed OS 5 default to a baud rate of 9600. Perhaps we should reflect that here as well since this is mainly used with mbed OS? Is there another reason (besides preference, speed, or that's what it was before) that this is 115200?